src/cli-validator/index.js

```diff
@@ -6,7 +6,7 @@
 require('./utils/checkVersion')('8.9.0');
 require('./utils/updateNotifier');
 
-const program = require('commander');
+const program = require('./utils/modified-commander.js');
```

Comment:
The `.js` isn't necessary here, just a Node convention to not use it
```suggestion
const program = require('./utils/modified-commander');
```